exporter/clickhousetracesexporter/clickhouse_exporter.go

```diff
@@ -403,6 +405,16 @@ func newStructuredSpan(otelSpan ptrace.Span, ServiceName string, resource pcommo
 
 // traceDataPusher implements OTEL exporterhelper.traceDataPusher
 func (s *storage) pushTraceData(ctx context.Context, td ptrace.Traces) error {
+	// if the new schema is enabled don't write to the old tables
+	err := s.pushTraceDataV3(ctx, td)
+	if err != nil {
+		return err
+	}
+	// if new schema is forced exit here else write to the old tables as well.
+	if s.useNewSchema {
+		return nil
+	}
```

Comment:
What is the `if the new schema is enabled` vs `if new schema is forced`?

Reply:
there is only one flag `use_new_schema`

* if **use_new_schema**: false (default) then it will write to old tables as well the new ones.
* If  **use_new_schema**: true , then it will just write to the new one. So we are returning here.